plugins/outputs/newrelic/newrelic.go

```diff
@@ -51,14 +57,16 @@ func (nr *NewRelic) Connect() error {
 	if nr.InsightsKey == "" {
 		return fmt.Errorf("InsightKey is a required for newrelic")
 	}
+	//TODO init nr.client here
+	nr.getHTTPClient()
```

Comment:
The function returns `error` and it really might do so. Ignoring those errors is extremely bad! Either move the code of the function up here _or_ check the return code!

Reply:
Thanks for catching this, my mistake.

Corrected.